zookeeper.out

```diff
@@ -0,0 +1,31 @@
+2017-10-05 20:21:32,418 [myid:] - INFO  [main:QuorumPeerConfig@124] - Reading configuration from: /Users/paulrogers/bin/zookeeper-3.4.9/bin/../conf/zoo.cfg
```

Comment:
I guess this file was added by mistake...

Reply:
Yup. Removed.